moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -269,6 +269,15 @@ class MoveGroupInterface
       If the value is greater than 1, it is set to 1.0. */
   void setMaxAccelerationScalingFactor(double max_acceleration_scaling_factor);
 
+  /** \brief Limit the maximum cartesian speed for a given link.
+      The unit of the speed is meters per second and must be greater than 0.
```

Comment:
```suggestion
      The unit of the speed is meter per second and must be greater than 0.
```
at least that would be consistent ... not sure what is correct actually